src/app/static/src/app/components/plots/Filters.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div>
+        <h4 v-translate="'filters'"></h4>
+        <div :id="'filter-' + filter.id" v-for="filter in filters" class="form-group">
+            <filter-select :value="getSelectedFilterValues(filter.id)"
+                           :multiple="filterIsMultiple(filter.id)"
+                           :label="filter.label"
+                           :options="filter.options"
+                           :disabled="filter.options.length==0"
+                           @select="onFilterSelect(filter, $event)"></filter-select>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import FilterSelect from "./FilterSelect.vue";
+    import {Dict, Filter} from "../../types";
+    import {FilterOption} from "../../generated";
+
+    interface Props {
+        filters: Filter[],
+        selectedFilterOptions: Dict<FilterOption[]>,
+        selectMultipleFilterIds: string[]
```

Comment:
Could we just add `allowMultiple` as a field on `Filter`?

Reply:
Yep. Whether multiple values should be selectable for a filter depends on the (output) plot - you can select multiple values for other filters in the barchart. So I was sort of seeing Filters as immutable things which come from the backend and can be presented with different functionality in different plots. However, we do actually have different filters getters for each plot type as there was initially the idea that you might want to have different filters per plot. So I could make a subclass of Filter which includes allowMultiple and return that in the getter for the choropleth filters, allowing multiple for area but not the others.